packages/python-google-compute-engine/google_compute_engine/service_utils.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
```

Comment:
Instead of defining a new module for this, please move the logic into the module where it's needed.

Reply:
done